cmake/modules/FindGnuLd.cmake

```diff
@@ -43,10 +43,10 @@ else()
 
   find_program(GNULD_LINKER ${CROSS_COMPILE}ld.bfd ${LD_SEARCH_PATH})
   if(GNULD_LINKER)
-    set(GNULD_LINKER_IS_BFD TRUE)
+    set(GNULD_LINKER_IS_BFD ON CACHE BOOL "Is linker ld.bfd compatible")
   else()
     find_program(GNULD_LINKER ${CROSS_COMPILE}ld ${LD_SEARCH_PATH})
-    set(GNULD_LINKER_IS_BFD FALSE)
+    set(GNULD_LINKER_IS_BFD OFF CACHE BOOL "Is linker ld.bfd compatible")
```

Comment:
```suggestion
    set(GNULD_LINKER_IS_BFD OFF CACHE BOOL "Is linker ld.bfd compatible" FORCE)
```

Reply:
I added the code to return early if GNULD_LINKER_IS_BFD is already set. So no need to FORCE a change.